src/Components/Album.css

```diff
@@ -0,0 +1,109 @@
+
+.album-container {
+ width: 100%;
+}
+
+.album {
+padding: 15px;
+ margin: 20px;
+ padding-bottom: 30px;
+}
+
+  .cover-image-container {
+    position: relative;
+  }
+
+  .cover-image {
+    width: 100%;
+  }
+
+
+  .icon-container {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom:0;
+    left: 0;
+    background-color: rgba(0, 0, 0, 0.6); /* When you hover over an album, it should go darker */
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    opacity: 0;
+    transition: opacity 0.3s ease-in-out;
+  }
+
+  .album:hover .icon-container {
+    opacity: 1;
+  }
+
+  /* icons over the album when hovering*/
```

Comment:
Is this comment reffering to the code snippet above? then maybe move it to line 34